resources/lib/svt.py

```diff
@@ -333,15 +333,15 @@ def getItems(section_name, page):
 
   returned_items = []
   for video in json_data["data"]:
-    is_program = video.get("hasEpisodes", False)
+    is_program = video.get("episodic", False)
```

Comment:
Unrelated to this issue but seems to have been renamed (maybe not needed anymore if versions are used?)

Reply:
Good find!
I will see if this is needed anymore.